athena_cfn/athena-cfn.yml

```diff
@@ -0,0 +1,146 @@
+    Default: query
+    NoEcho: false
+    Description: Suffix for Quilt-specific Athena workgroup
+    Type: String
+    MinLength: 3
+    AllowedPattern: '[a-z][a-z0-9]*'
+
+# Non-parameters (no intrinsics allowed)
+# ARNDatacatalog: !Sub 'arn:aws:glue:${AWS::Region}:${AWS::AccountId}:datacatalog/*'
+# ARNCatalog:!Sub 'arn:aws:glue:${AWS::Region}:${AWS::AccountId}:catalog'
+
+Resources:
+  QuiltAthenaOutput:
+    Type: "AWS::S3::Bucket"
+    #DeletionPolicy: Retain
+    Properties:
+      BucketName: !Join ["-", [!Ref AWS::StackName, !Ref SuffixWorkGroup, !Ref SuffixBucket]]
+
+  QuiltAthenaWorkgroup:
+    Type: AWS::Athena::WorkGroup
+    DeletionPolicy: Delete
+    Properties:
+      Name: !Join ["-", [!Ref AWS::StackName, !Ref SuffixWorkGroup]]
+      RecursiveDeleteOption: true
+      WorkGroupConfiguration:
+        PublishCloudWatchMetricsEnabled: true
+        ResultConfiguration:
+          OutputLocation: !Join ["", ["s3://" , Ref: QuiltAthenaOutput, "/"]]
+
+  QuiltAthenaAccess:
+    DeletionPolicy: Delete
+    Type: AWS::IAM::ManagedPolicy
+    Properties:
+      Path: /
+      PolicyDocument:
+        Version: '2012-10-17'
+        Statement:
+        - Sid: AccessAllAthenaWorkgroups
+          Effect: Allow
+          Action: athena:ListWorkGroups
+          Resource: '*'
+        - Sid: AccessQuiltAthenaWorkgroup
+          Effect: Allow
+          Action:
+          - athena:Create*
+          - athena:*Get*
+          - athena:List*
+          - athena:StartQueryExecution
+          Resource:
+          - !Sub 'arn:aws:glue:${AWS::Region}:${AWS::AccountId}:catalog'
+          - !Sub 'arn:aws:glue:${AWS::Region}:${AWS::AccountId}:datacatalog/*'
+          - !Sub
+            - 'arn:aws:athena:${AWS::Region}:${AWS::AccountId}:workgroup/${Workgroup}'
+            - {Workgroup: !Ref QuiltAthenaWorkgroup}
+
+        - Sid: AccessGlueDatabases
+          Effect: Allow
+          Action:
+          - glue:GetDatabase
+          - glue:GetDatabases
+          - glue:GetTable
+          - glue:GetTables
+          Resource: '*'
+        - Sid: ModifyGlueDatabases
+          Effect: Allow
+          Action:
+          - glue:CreateTable
+          - glue:DeleteTable
+          - glue:UpdateTable
+          Resource:
+          - !Sub 'arn:aws:glue:${AWS::Region}:${AWS::AccountId}:catalog'
+          - !Sub
+            - 'arn:aws:glue:${AWS::Region}:${AWS::AccountId}:database/${Database}'
+            - {Database: !Ref DatabaseName}
+          - !Sub
+            - 'arn:aws:glue:${AWS::Region}:${AWS::AccountId}:table/${Database}/${Tables}'
+            - {Database: !Ref DatabaseName, Tables: !Ref DatabaseTableRegex}
+        - Sid: AccessQuiltAthenaOutputBucket
+          Effect: Allow
+          Action:
+          - s3:GetBucketLocation
+          - s3:GetObject
+          - s3:ListBucket
+          - s3:ListBucketMultipartUploads
+          - s3:ListMultipartUploadParts
+          - s3:AbortMultipartUpload
+          - s3:CreateBucket
```

Comment:
`s3:CreateBucket` seems unneeded since bucket is already created by this template.
```suggestion
```